pom.xml

```diff
@@ -9,32 +9,39 @@
 	</parent>
 	
 	<groupId>eu.arrowhead</groupId>	
-	<artifactId>client-skeleton</artifactId>
-	<version>4.1.3.3</version>
+	<artifactId>application-skeleton-java-spring</artifactId>
+	<version>4.4.0.0</version>
 	<packaging>pom</packaging>
-	<name>client-skeleton-java-spring</name>
-	<description>Arrowhead Client Skeleton using Spring Boot</description>
+	<name>application-skeleton-java-spring</name>
+	<description>Arrowhead Application Skeleton using Spring Boot</description>
 	
 	<url>https://github.com/arrowhead-f/client-java-spring</url>
 	<licenses>
     	<license>
-	     	<name>Apache License, Version 2.0</name>
-	     	<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+	     	<name>Eclipse Public License - v 2.0</name>
+	     	<url>https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt</url>
 	   </license>
   	</licenses>
 	
 	<modules>
-	    <module>client-skeleton-consumer</module>
-	    <module>client-skeleton-provider</module>
-	    <module>client-skeleton-subscriber</module>
-	    <module>client-skeleton-publisher</module>
+	    <module>application-skeleton-consumer</module>
+	    <module>application-skeleton-provider</module>
+	    <module>application-skeleton-subscriber</module>
+	    <module>application-skeleton-publisher</module>
   	</modules>
-		
+	
+	<repositories>
+		<repository>
+			<id>ossrh</id>
+			<url>https://s01.oss.sonatype.org/content/repositories/aiaitia-1017</url>
+		</repository>
+	</repositories>
```

Comment:
is this necessary after the application-library will be available in maven central?

Reply:
ok, I see your comment now.